low-level/files/opt/seagate/sspl/setup/sspl_post_install.py

```diff
@@ -0,0 +1,168 @@
+        # node replacement case
+        # Onward LDR_R2, consul will be abstracted out and it won't exit as hard dependeny of SSPL
+        if PRODUCT_NAME == "LDR_R1":
+            # setup consul if not running already
+            if not os.path.exists(REPLACEMENT_NODE_ENV_VAR_FILE):
+                sspl_setup_consul = f"{self._script_dir}/sspl_setup_consul -e {ENVIRONMENT}"
+                output, error, returncode = SimpleProcess(sspl_setup_consul).run()
+                if returncode != 0:
+                    raise PostInstallError(returncode, error, sspl_setup_consul)
+
+        # Install packages which are not available in YUM repo, from PIP
+        pip_cmd = f"python3 -m pip install -r {SSPL_BASE_DIR}/low-level/requirements.txt"
+        output, error, returncode = SimpleProcess(pip_cmd).run()
+        if returncode != 0:
+            raise PostInstallError(returncode, error, pip_cmd)
+        # Splitting current function into 2 functions to reduce the complexity of the code.
+        self.make_file_operations(PRODUCT, ENVIRONMENT, RMQ_CLUSTER)
+
+    def make_file_operations(self, PRODUCT, ENVIRONMENT, RMQ_CLUSTER):
+        # NOTE: By default the sspl default conf file will not be copied.
+        # The provisioner is supposed to copy the appropriate conf file based
+        # on product/env and start SSPL with it.
+        # TODO: Disable this default copy once the provisioners are ready.
+        if not os.path.exists(file_store_config_path):
+            shutil.copyfile(f"{SSPL_BASE_DIR}/conf/sspl.conf.{PRODUCT}", file_store_config_path)
+
+        # For Dev environment
+        if ENVIRONMENT == "DEV":
+            shutil.copyfile(f"{SSPL_BASE_DIR}/conf/sspl.conf.{PRODUCT}", file_store_config_path)
+
+        # Copy rsyslog configuration
+        if not os.path.exists(self.RSYSLOG_CONF):
+            shutil.copyfile(f"{SSPL_BASE_DIR}/low-level/files/{self.RSYSLOG_CONF}", self.RSYSLOG_CONF)
+
+        if not os.path.exists(self.RSYSLOG_SSPL_CONF):
+            shutil.copyfile(f"{SSPL_BASE_DIR}/low-level/files/{self.RSYSLOG_SSPL_CONF}", self.RSYSLOG_SSPL_CONF)
+
+        # Create soft link for SINGLE product name service to existing LDR_R1, LDR_R2 service
+        # Instead of keeping separate service file for SINGLE product with same content.
+        currentProduct = f"{SSPL_BASE_DIR}/conf/sspl-ll.service.{PRODUCT}"
+        if (PRODUCT == "SINGLE" and not os.path.exists(currentProduct)) or (PRODUCT == "DUAL" and not os.path.exists(currentProduct)):
+            os.symlink(f"{SSPL_BASE_DIR}/conf/sspl-ll.service.{PRODUCT_NAME}", currentProduct)
+
+        if PRODUCT == "CLUSTER" and not os.path.exists(currentProduct):
+            os.symlink(f"{SSPL_BASE_DIR}/conf/sspl-ll.service.LDR_R2", currentProduct)
+
+        # Copy sspl-ll.service file and enable service
+        shutil.copyfile(currentProduct, "/etc/systemd/system/sspl-ll.service")
+        Utility.enable_disable_service(service='sspl-ll.service', action='enable')
```

Comment:
code from cortx-utils service class dbus handler apis only we are supposed to use, remove deuplicated from sspl utility module, as per Ujjwal's comments in morning

Reply:
Done